lib/src/se/llbit/chunky/resources/SettingsDirectory.java

```diff
@@ -106,6 +106,13 @@ public static File getWorkingDirectory() {
   }
 
   /**
+   * @return the plugin directory
+   */
+  public static File getPluginsDirectory() {
+    return new File(getWorkingDirectory(), "plugins");
```

Comment:
Is there a particular reason you used `getWorkingDirectory()` rather than `getSettingsDirectory()` here? It seems like the settings directory is what you intended to use here.